src/Helper/Route.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
The `send` helper already exists in the `Request` namespace with automatic header controls, that can also be used outside the callback functions.

Reply:
Yes I understand .. I never meant to edit that file .. probably when i tried viewing it